.storybook/preview-head.html

```diff
@@ -1,8 +1,3 @@
-<link
-  href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&amp;display=swap"
-  rel="stylesheet"
-/>
```

Comment:
Looks like we no longer use Poppins as of https://github.com/BuoySoftware/anchor-ui/commit/9c25967, so cleaning this up.